contrib/completions/bash/openshift

```diff
@@ -5357,6 +5425,7 @@ _openshift_admin()
     commands+=("create-bootstrap-policy-file")
     commands+=("create-login-template")
     commands+=("create-provider-selection-template")
+    commands+=("create-error-template")
```

Comment:
The only inputs to the error template are error code and error message... don't see why we wouldn't reuse the template for more things in the future